tests/bud.bats

```diff
@@ -3981,6 +3981,36 @@ _EOF
   run_buildah rmi myalpine
 }
 
+@test "build verify cache behaviour with --cache-ttl" {
+  mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
+
+  cat > ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile1 << _EOF
+FROM alpine
+RUN touch hello
+RUN echo world
+_EOF
+
+  run_buildah build $WITH_POLICY_JSON --layers -t source -f ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile1
+  # Sleep for 1s to verify that we are not
+  # going to use cached layers.
+  sleep 1s
```

Comment:
It'd be simpler to use the `--timestamp` flag to force the image creation timestamp to be some point in the past here.

Reply:
Yes created with `--timestamp` somewhere in `2021`